week-1/Homework/extra/1-currency-conversion.js

```diff
@@ -4,8 +4,10 @@
   The business is breaking out into a new market and need to convert prices to USD
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
-
-function convertToUSD() {}
+console.clear();
```

Comment:
Hey @AKaramifar , can you explain the use of `console.clear` here?

Reply:
because I need to clean all outputs in console and when I run the code I see only last output